### PR TITLE
Fix for issue#533

### PR DESCRIPTION
**Pull request by marcin-krolik**
Fixed call for list of available API versions for Cinder.

### CODE REVIEW

File: openstack/blockstorage/v1/apiversions/urls.go [hunk 1]
```diff
@@ -11,5 +12,7 @@ func getURL(c *gophercloud.ServiceClient, version string) string {
 }
 
 func listURL(c *gophercloud.ServiceClient) string {
-	return c.ServiceURL("")
+	u, _ := url.Parse(c.ServiceURL(""))
+	u.Path = "/"
+	return u.String()
```
Comment (jrperritt) [inline openstack/blockstorage/v1/apiversions/urls.go]: Try just returning `c.Endpoint` such as [here](https://github.com/rackspace/gophercloud/blob/master/openstack/orchestration/v1/apiversions/urls.go#L6) and [here](https://github.com/rackspace/gophercloud/blob/master/openstack/networking/v2/apiversions/urls.go#L10) and see if that works.

Comment (marcin-krolik) [inline openstack/blockstorage/v1/apiversions/urls.go]: Problem is ```c.Endpoint``` contains full service URL with path included
like ```http://public.fuel.local:8776/v2/d98e06adf5db49ad9f372625cad7840b/``` 
while for list of avaiable API version one needs to call root path 
like ```http://public.fuel.local:8776/```

Comment (jrperritt) [inline openstack/blockstorage/v1/apiversions/urls.go]: Ah, I see now in the service catalog.

**Comment by jrperritt** [comment]
LGTM. +2

**Pull request merged**